### PR TITLE
add: Persian Language minor

### PR DESCRIPTION
**Pull request by windsornguyen**
**References**
- Linear: [DEV-80](https://linear.app/hoagie/issue/DEV-80/persian-language-minor)

**Proposed Changes**
- Added [Persian Language Minor](https://ua.princeton.edu/fields-study/minors/persian-language)

### CODE REVIEW

File: backend/minors/PER.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        max_counted: 1
+        min_needed: 1
+        explanation: A fifth Near Eastern Studies course (either an additional language class or a content class listed or cross-listed in Near Eastern Studies).
+        course_list:
+          - NES ***
+      - name: Senior Translation
+        max_counted: 1
+        min_needed: 1
+        explanation: |-
+          At the end of their language study at Princeton, a student minoring in Persian Language must work to submit a polished translation from a work in Persian in order to complete the requirements of the minor. This translation could be from a work of literature (e.g., an excerpt of a prose work, a poem, a short story), an essay, a film, a song or another piece approved by NES language faculty. Translations must be submitted to the director of undergraduate studies before Dean's Date in the student's final semester. This submission should also include a list of the student's departmental and language courses and any other courses or experience that might be relevant to the minor, such as study abroad or internships related to the Near East.
```
Comment (issacli-0821) [inline backend/minors/PER.yaml:46]: Let's add an "Editor's note" here

**Review by issacli-0821 — CHANGES_REQUESTED** [review]
One comment and we should be good

File: backend/minors/PER.yaml (hunk 1 above)
Comment (windsornguyen) [reply backend/minors/PER.yaml]: committed #2e96aea to address this

**Review by issacli-0821 — APPROVED** [review]
Looks good!

**Pull request merged**